readme.md

```diff
@@ -61,30 +61,6 @@ It shows how to connect a device, handle disconnect events, cases to handle when
 
 If you would like to build or change the SDK source code, please follow the [devguide](doc/devguide.md).
 
-## OS platforms and hardware compatibility
-
-> .NET Standard 1.3 (IoT Hub SDKs only) is last supported in the [2020-02-27](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2020-2-27) and in the [2020-1-31 LTS](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-1-31) releases.
-
-The IoT Hub device SDK for .NET can be used with a broad range of device platforms and is officially supported on the following Operating Systems:
-
-*  Windows versions officially supported by Microsoft.
-*  [Linux distributions](https://docs.microsoft.com/en-us/dotnet/core/install/linux) supported by .NET core.
-
-> Note: For Linux, we test our clients against Ubuntu 16.04.7 LTS.
-
-The NuGet packages provide support for the following .NET flavors:
-- .NET Standard 2.1
-- .NET Standard 2.0
-- .NET Framework 4.7.2 (IoT Hub SDKs only)
-- .NET Framework 4.5.1 (IoT Hub SDKs only)
-
-For details on .NET support see the [.NET Standard documentation](https://docs.microsoft.com/dotnet/standard/net-standard).
-For details on OS support see the following resources:
-
-- [.NET Core Runtime ID Catalog](https://docs.microsoft.com/dotnet/core/rid-catalog)
```

Comment:
looks like we lost this and the below links in the new doc

Reply:
I don't follow why we had these other links within our README in the first place. Do you know why we need them around still?

Reply:
- afaik rid are used to specify specific platforms that applications resolve to, i.e. I can compile a comsole app a different way for an x64 system vs an x86 system. I don't remember seeing such references in our csproj file though, so it might just be legacy code from a time where we defined projects differently.
- the system requirements would be useful to highlight here, since that is the min req for using our sdks
- the TLS specifications also need to be highlighted on our landing page (esp with all certs related changes that seem to be coming in the future)

Reply:
I can add the TLS link and the system requirements link then, sure. I don't see any references to the `<RuntimeIdentifier>` tag that the rid document outlines, so I think we are safe to remove that link though

Reply:
That sounds good